### PR TITLE
kcm-fcitx5: new package

### PR DESCRIPTION
**Pull request by eatradish**
https://gitlab.com/fcitx/kcm-fcitx5

### CODE REVIEW

File: extra-i18n/kcm-fcitx5/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=kcm-fcitx5
+PKGDES="KDE Config Module for Fcitx5"
+PKGDEP="fcitx5 knewstuff kcmutils"
```
Comment (liushuyu) [inline extra-i18n/kcm-fcitx5/autobuild/defines]: `PKGSEC`?

**Review by liushuyu — CHANGES_REQUESTED** [review]
`PKGSEC`?

File: extra-i18n/kcm-fcitx5/autobuild/defines (hunk 1 above)
Comment (eatradish) [reply extra-i18n/kcm-fcitx5/autobuild/defines]: > `PKGSEC`?

OK...wait

Comment (eatradish) [reply extra-i18n/kcm-fcitx5/autobuild/defines]: > `PKGSEC`?

ok...added

**Comment by MingcongBai** [comment]
Looks good to me. Thank you!

**Pull request merged**